### PR TITLE
added some minor logging, updated log level for failure cases

### PR DESCRIPTION
**Pull request by tbenz9**
Added some basic logging for failure cases.  Modified log level.

### CODE REVIEW

**Comment by ofaaland** [comment]
@tbenz9 thank you.  You and I know the justification for changing the log level to info and the motivation for making these changes because we discussed them, but can you summarize those in the commit message?  Thank you.

File: scripts/lustre [hunk 1]
```diff
@@ -54,15 +54,17 @@ lustre_monitor() {
...
 
     for mntdev_path in $mntdevs ; do
         mntdev=$(cat $mntdev_path 2>/dev/null)
         if [ $mntdev = $OCF_RESKEY_dataset ] ; then
-        ocf_log debug "$OCF_RESKEY_dataset found in $mntdev_path"
-        return $OCF_SUCCESS
+            ocf_log debug "$OCF_RESKEY_dataset found in $mntdev_path"
+            return $OCF_SUCCESS
         fi
+        ocf_log info "$mntdev is not in $OCF_RESKEY_dataset"
```
Comment (ofaaland) [inline scripts/lustre]: wording is backwards.  Instead how about "$OCF_RESKEY_dataset not found in $mntdev_path"?

Comment (ofaaland) [reply scripts/lustre]: ping

Comment (tbenz9) [reply scripts/lustre]: @ofaaland I made the change last week, resolved this thread and requested a new code review.  See this commit: 7ecb78865fb3ebb5fb8b654f9c88ba1a0ac60bd1

Comment (ofaaland) [reply scripts/lustre]: Oops, sorry I overlooked that.

**Comment by tbenz9** [comment]
Done, I think.  This was my first time attempting a CLI squash... :)

**Comment by ofaaland** [comment]
@tbenz9  may I add your "Signed-off-by:  " to the commit message?  It indicates that you agree to:
https://developercertificate.org/

**Comment by tbenz9** [comment]
Yes, of course.  I read the link, I agree to all.

**Comment by ofaaland** [comment]
Updated commit message, merged as
* 248d753 Added some minor logging, updated log level for failure cases
